impala/hiveserver2.py

```diff
@@ -1585,23 +1593,27 @@ def build_summary_table(summary, idx, is_fragment_root, indent_level, output):
         setattr(max_stats, attr, 0)
 
     node = summary.nodes[idx]
-    for stats in node.exec_stats:
-        for attr in attrs:
-            val = getattr(stats, attr)
-            if val is not None:
-                setattr(agg_stats, attr, getattr(agg_stats, attr) + val)
-                setattr(max_stats, attr, max(getattr(max_stats, attr), val))
-
-    if len(node.exec_stats) > 0:
+    if node.exec_stats is not None:
+        for stats in node.exec_stats:
+            for attr in attrs:
+                val = getattr(stats, attr)
+                if val is not None:
+                    setattr(agg_stats, attr, getattr(agg_stats, attr) + val)
+                    setattr(max_stats, attr, max(getattr(max_stats, attr), val))
+
+    if node.exec_stats is not None and node.exec_stats:
```

Comment:
wouldn't be simply if node.exec_stats: enough?

Reply:
Replaced this with a new exec_summary.py from https://github.com/apache/impala/commit/e73e2d40da115ed3804ffaecc0850c853b0e6330